src/app/page.tsx

```diff
@@ -13,20 +13,10 @@ const BUTTON_STATE_MAP: Record<AttendResult["type"], AttendButtonState> = {
   SUCCESS: "DONE",
   FAILURE: "OVERTIME",
 };
-function attendApp() {
-  // useAttendAction フックを使用して状態とアクションを管理
-  const [attendResult, submitAttendAction] = useAttendAction();
-  const buttonState = BUTTON_STATE_MAP[attendResult["type"]];
 
-  return (
-    <div>
-      <AttendButton state={buttonState} onClick={submitAttendAction} />
-    </div>
-  );
-}
 export default function Home() {
-  const attendState = "ENABLED";
-
+  const [attendResult, submitAttendAction] = useAttendAction();
+  const attendState = BUTTON_STATE_MAP[attendResult["type"]];
```

Comment:
```suggestion
  const attendState = BUTTON_STATE_MAP[attendResult.type];
```
こう書いても同じなので, こう短く書いておいたほうが良いかと思います.